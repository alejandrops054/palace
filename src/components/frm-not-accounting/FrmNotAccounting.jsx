/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/

import React from 'react';
import { SimpleTable } from "../index";
const FrmNotAccounting = () => {
    const columns = [
        {
            title: 'provision',
            dataIndex: 'provision',
            key: 'provision',
            width: "1%",
        },
        {
            title: 'Application',
            dataIndex: 'application',
            key: 'application',
            width: "1%",
        },
        {
            title: 'Folio',
            dataIndex: 'folio',
            key: 'folio',
            width: "1%",
        },
        {
            title: 'Date',
            dataIndex: 'date',
            key: 'date',
            width: "1%",
        },
        {
            title: 'Port Folio',
            dataIndex: 'portfolio',
            key: 'portfolio',
            width: "1%",
        },
        
    ];
    const data = [
        {provision:'', application: '',folio:'', date:'', portfolio:''},

    ];
    return (
        <div className="card">
            <div className="card-body">
                <SimpleTable columns={columns} data={data} type="table" />
            </div>
        </div>
    );
};

export default FrmNotAccounting;
