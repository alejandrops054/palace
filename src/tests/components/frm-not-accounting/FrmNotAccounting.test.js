/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/

import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import FrmNotAccounting from '../../../components/frm-not-accounting/FrmNotAccounting'

describe('FrmNotAccounting component', () => {
    test('debe mostrar snapshot', () => {

        const wrapper = shallow(<FrmNotAccounting />)
        expect(wrapper).toMatchSnapshot();
    })

})