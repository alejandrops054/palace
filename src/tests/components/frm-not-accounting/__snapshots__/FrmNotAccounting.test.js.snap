// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FrmNotAccounting component debe mostrar snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card-body"
  >
    <SimpleTable
      columns={
        Array [
          Object {
            "dataIndex": "provision",
            "key": "provision",
            "title": "provision",
            "width": "1%",
          },
          Object {
            "dataIndex": "application",
            "key": "application",
            "title": "Application",
            "width": "1%",
          },
          Object {
            "dataIndex": "folio",
            "key": "folio",
            "title": "Folio",
            "width": "1%",
          },
          Object {
            "dataIndex": "date",
            "key": "date",
            "title": "Date",
            "width": "1%",
          },
          Object {
            "dataIndex": "portfolio",
            "key": "portfolio",
            "title": "Port Folio",
            "width": "1%",
          },
        ]
      }
      data={
        Array [
          Object {
            "application": "",
            "date": "",
            "folio": "",
            "portfolio": "",
            "provision": "",
          },
        ]
      }
      type="table"
    />
  </div>
</div>
`;
